src/pages/signup.js

```diff
@@ -199,6 +214,18 @@ const Signup = () => {
           </Stack>
         </Form>
       </Column>
+      <Modal
+        open={open}
+        passiveModal
+        
+        onRequestClose={() => navigate("/login")}
+      >
+        <Stack gap={7}>
+              <h4>Successfull signed in!</h4>
```

Comment:
Since we are signing up we can improve this message to "Successful sign up" Or "registration complete" or something much more explicit :)